internal/adapters/operation/operation_storage_redis.go

```diff
@@ -365,3 +365,31 @@ func buildOperationFromMap(opMap map[string]string) (*operation.Operation, error
 		ExecutionHistory: executionHistory,
 	}, nil
 }
+
+func (r *redisOperationStorage) CleanExpiredOperations(ctx context.Context, schedulerName string) error {
+	operationsIDs, err := r.client.ZRangeByScore(ctx, r.buildSchedulerHistoryOperationsKey(schedulerName), &redis.ZRangeBy{
+		Min: "-inf",
+		Max: "+inf",
+	}).Result()
+
+	if len(operationsIDs) > 0 {
```

Comment:
@mflilian Before proceeding to the if below, we should check and treat if this method returns error. We have defined the `err` var but we aren't checking it.